assets/js/src/core/modules/element/dynamic-types/definitions/objects/data-related/components/many-to-many-relation/hooks/use-value.ts

```diff
@@ -33,22 +38,47 @@ interface UseValueReturn {
   addItems: (items: ManyToManyRelationValueItem[]) => void
   addAssets: (assets: Asset[]) => Promise<void>
   maxRemainingItems?: number
+  pathFormatterClass?: string | null
```

Comment:
Currently we have the loading spinner + the triggered `format-path` request also when no path formatter is configured. We need to find a way to skip the path formatter logic when no `pathFormatterClass` is defined.